src/quo2/components/list_items/contact_item.cljs

```diff
@@ -0,0 +1,56 @@
+(ns quo2.components.list-items.contact-item
+  (:require [re-frame.core :as re-frame]
+            [status-im.utils.handlers :refer [<sub]]
+            [quo2.foundations.typography :as typography]
+            [quo2.components.icon :as icons]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.avatars.user-avatar :as user-avatar]
+            [quo.react-native :as rn]
+            [status-im.utils.utils :refer [get-shortened-address]]
+            [quo.platform :as platform]))
+
+(defn open-chat [chat-id]
+  (re-frame/dispatch [:dismiss-keyboard])
```

Comment:
I'm not sure anymore of the decision, but I remember @flexsurfer using `rf` as the require alias that we should be using. I like it because re-frame is so pervasive in the codebase that I think everybody can quickly internalize what `rf` means.

Reply:
Actually, it should be `<sub`.